tools/python/util/convert_onnx_models_to_ort.py

```diff
@@ -258,12 +258,10 @@ def parse_args():
         "processed.",
     )
 
-    return parser.parse_args()
+    return parser.parse_args() if args is None else parser.parse_args(args)
```

Comment:
this should work, as the default value of args is None
```suggestion
    parser.parse_args(args)
```